platforms/vmware/variables.tf

```diff
@@ -114,6 +114,19 @@ variable "tectonic_vmware_etcd_clusters" {
 EOF
 }
 
+variable "tectonic_vmware_etcd_resource_pool" {
+  type = "map"
+
+  description = <<EOF
+  terraform map of etcd node(s) vSphere Resource Pools, example:
+  tectonic_vmware_etcd_clusters = {
```

Comment:
please update all examples to be _resource_pool not _clusters for: etcd, masters & workers